pkg/splunk/enterprise/util.go

```diff
@@ -1667,6 +1433,31 @@ func checkIfFileExistsOnPod(c splcommon.ControllerClient, namespace string, podN
 	return fileTestResult == 0
 }
 
+// TODO: gaurav - change this API to use new podExecClient
+// createDirOnSplunkPods creates the required directory for the pod/s
+func createDirOnSplunkPods(c splcommon.ControllerClient, cr splcommon.MetaObject, replicas int32, path string) error {
```

Comment:
WIll change this API when rebasing https://github.com/splunk/splunk-operator/pull/662 to the latest feature branch after merging this PR.